src/ledger/NetworkConfig.h

```diff
@@ -100,6 +100,22 @@ struct InitialSorobanNetworkConfig
     static constexpr uint32_t LEDGER_MAX_TX_COUNT = 10;
 };
 
+struct MinimumSorobanNetworkConfig
```

Comment:
There is no need to prefix every value with `MIN_` here. It also would be better if you named the fields in the same way as in `InitialNetworkConfig`.